packages/react/src/components/ExpandCollapsePanel/PanelTrigger.tsx

```diff
@@ -17,6 +19,8 @@ function PanelTrigger({
   open,
   fullWidth,
   onClick,
+  expandedIcon = 'chevron-down',
+  collapsedIcon = 'chevron-right',
```

Comment:
Very minor nitpick: `iconExpanded` and `iconCollapsed` would be a cleaner way to define this since you're defining the icon for the expanded/collapsed state.

Reply:
done